Plate 4 Image-based Analysis (#31)
WayScience/CFReT_data: 3.process_cfret_features/README.md

# 3. Processing Extracted Single Cell and Image Features from CellProfiler

In this module, we present our pipeline for processing outputted `.sqlite` files with single cell and image features from CellProfiler.

> The processed features are saved into `parquet` files for use during analysis for each plate.

Reviewer: ```suggestion
The processed features are saved into `parquet` files, which are used for analysis for each plate.
```
I think this is what you mean? Feel free to reject if it is incorrect.

Author: Yes that is what I meant and sounds clearer, thanks!

## CytoTable

We use [CytoTable] to convert the CellProfiler SQLite outputs into merged single-cell parquet files, where features from each compartment (e.g., nuclei, cells, and cytoplasm) are combined into one row per single-cell.

> The exact function we used is called `convert`, which you can find more info on in [the documentation](https://cytomining.github.io/CytoTable/python-api.html#module-cytotable.convert).

Reviewer: ```suggestion
The exact function we used is called `convert`, which you can find more info on in the [documentation](https://cytomining.github.io/CytoTable/python-api.html#module-cytotable.convert).
```

Author: 👍

## Pycytominer

We use [pycytominer](https://github.com/cytomining/pycytominer) to perform the aggregation, merging, and normalization of the CFReT single cell features.

> For more information regarding the functions that we used, please see [the documentation](https://pycytominer.readthedocs.io/en/latest/pycytominer.cyto_utils.html#pycytominer.cyto_utils.cells.SingleCells.merge_single_cells) from the Pycytominer team.

Author: 👍
